privacyidea/lib/resolvers/SQLIdResolver.py

```diff
@@ -489,7 +489,15 @@ def loadConfig(self, config):
         # Session should be closed on teardown
         register_finalizer(self.session.close)
         self.session._model_changes = {}
-        self.TABLE = Table(self.table, MetaData(), autoload_with=self.engine)
+
+        metadata = MetaData()
+        table_parts = self.table.split(".")
+        schema = table_parts[0] if len(table_parts) > 1 else None
+        self.table = table_parts[-1]
+        log.debug("Loading table {0!s} from schema {1!s}".format(self.table, schema))
+        if schema is not None:
```

Comment:
We can just pass `schema` to the `Table` constructor below:
https://docs.sqlalchemy.org/en/20/core/metadata.html#sqlalchemy.schema.Table.params.schema

Reply:
@plettich I've pushed a change to set the Table schema instead of setting it on the Metadata.

Reply:
The `if schema is not None` can be dropped completely since `None` is the default value for the `schema` parameter of `Table()`.